docs/src/pages/ui/primitives/menu/examples/controlledExample.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
```

Comment:
I love splitting these examples up into their own files!

Reply:
Yeah, it makes for a much more maintainable examples I think!